chainerrl/links/noisy_linear.py

```diff
@@ -27,7 +30,7 @@ def __init__(self, mu_link, sigma_scale=0.4):
 
         with self.init_scope():
             self.mu = L.Linear(in_size, self.out_size, self.nobias,
-                               initialW=LeCunUniform(1 / numpy.sqrt(3)))
+                               initialW=LeCunUniform(1 / self.xp.sqrt(3)))
```

Comment:
Could you explain the change to the line?

Reply:
The change is insignificant, either one can work. I was making a pass through the code, changing np to xp, but it's irrelevant. I can change back.

Reply:
Changed.